sqlglot/dialects/sqlite.py

```diff
@@ -29,6 +29,8 @@ class Parser(Parser):
         }
 
     class Generator(Generator):
+        NULL_TREATMENT_IMPLEMENTED = False
```

Comment:
what is this? this doesn't look used

Reply:
My bad. I should have removed it before submitting. It is a slip up from the previous revision.